### PR TITLE
[Data model] add expected times

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
@noferini  we could consider adding values for pions kaons and protons to simplify the workflows

**Comment by noferini** [comment]
it would make it easier indeed

File: Framework/Core/include/Framework/AnalysisDataModel.h [hunk 1]
```diff
@@ -266,6 +266,14 @@ DECLARE_SOA_EXPRESSION_COLUMN(DetectorMap, detectorMap, uint8_t, //! Detector ma
                                 ifnode(aod::track::trdPattern > (uint8_t)0, static_cast<uint8_t>(o2::aod::track::TRD), (uint8_t)0x0) |
                                 ifnode((aod::track::tofChi2 >= 0.f) && (aod::track::tofExpMom > 0.f), static_cast<uint8_t>(o2::aod::track::TOF), (uint8_t)0x0));
 
+DECLARE_SOA_DYNAMIC_COLUMN(TOFExpTime, tofExpTime, //! Expected time for the track to reach the TOF
+                           [](float length, float tofExpMom, float mMassZSqared) -> float {
+                             if (tofExpMom <= 0.f) {
+                               return -999.f;
+                             }
+                             return length * sqrt((mMassZSqared) + (tofExpMom * tofExpMom)) / (0.0299792f * tofExpMom);
```
Comment (jgrosseo) [inline Framework/Core/include/Framework/AnalysisDataModel.h]:
```suggestion
                             return length * std::sqrt((mMassZSqared) + (tofExpMom * tofExpMom)) / (0.0299792f * tofExpMom);
```

Comment (jgrosseo) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: Can we use one of those constants? https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Constants/include/CommonConstants/PhysicsConstants.h#L196 
If needed define the one you need: cm/ps?

Comment (njacazio) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: Indeed that was the best way to handle it, done!

**Comment by jgrosseo** [comment]
For me this is fine. @ddobrigk Do you have comments?

**Comment by ddobrigk** [comment]
Yes indeed, this can go! Thanks a lot!

**Review by jgrosseo — APPROVED** [review]

**Comment by njacazio** [comment]
@jgrosseo could it be that the CI is stuck?

**Comment by jgrosseo** [comment]
@ktf can we go ahead?

**Comment by ktf** [comment]
Full CI and slc7 work. Merging.

**Pull request merged**